pkgs/by-name/co/cosmic-session/package.nix

```diff
@@ -11,38 +11,33 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "cosmic-session";
-  version = "0-unstable-2024-01-17";
+  version = "1.0.0-alpha.1";
 
   src = fetchFromGitHub {
     owner = "pop-os";
-    repo = pname;
-    rev = "8e73c0f6940288c4a24a102a7ba9f20eb6bd754f";
-    sha256 = "sha256-plglQ9i+kcG70v9ElCzwNMhO1xcuEAQiO0DeZfRjbcg=";
+    repo = "cosmic-session";
+    rev = "epoch-${version}";
+    sha256 = "sha256-5zfEBNsMxtKPJZcGYZth/SoXrsg0gpug15VR5fPbvt0=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
       "cosmic-notifications-util-0.1.0" = "sha256-GmTT7SFBqReBMe4GcNSym1YhsKtFQ/0hrDcwUqXkaBw=";
-      "launch-pad-0.1.0" = "sha256-tnbSJ/GP9GTnLnikJmvb9XrJSgnUnWjadABHF43L1zc=";
+      "launch-pad-0.1.0" = "sha256-c+uawTQlg5SW8x7DOBG2Idv/AfIaCFNtLQLUz8ifT2I=";
     };
   };
 
   postPatch = ''
     substituteInPlace Justfile \
-        --replace '#!/usr/bin/env' "#!$(command -v env)" \
-        --replace 'target/release/cosmic-session' "target/${
-          rust.lib.toRustTargetSpecShort stdenv.hostPlatform
-        }/release/cosmic-session"
+      --replace-fail '{{cargo-target-dir}}/release/cosmic-session' 'target/${rust.lib.toRustTargetSpecShort stdenv.hostPlatform}/release/cosmic-session'
     substituteInPlace data/start-cosmic \
-        --replace '#!/bin/bash' "#!${lib.getBin bash}/bin/bash" \
-        --replace '/usr/bin/cosmic-session' "$out/bin/cosmic-session" \
-        --replace '/usr/bin/dbus-run-session' "${
-          lib.getBin dbus
-        }/bin/dbus-run-session"
-    substituteInPlace data/cosmic.desktop --replace '/usr/bin/start-cosmic' "$out/bin/start-cosmic"
+      --replace-fail '/usr/bin/cosmic-session' "${placeholder "out"}/bin/cosmic-session" \
+      --replace-fail '/usr/bin/dbus-run-session' "${lib.getBin dbus}/bin/dbus-run-session"
+    substituteInPlace data/cosmic.desktop --replace '/usr/bin/start-cosmic' "${placeholder "out"}/bin/start-cosmic"
```

Comment:
```suggestion
    substituteInPlace data/cosmic.desktop --replace-fail '/usr/bin/start-cosmic' "${placeholder "out"}/bin/start-cosmic"
```

Reply:
Thanks! I broke the line to be in line with the other lines.
That makes it easier to see.